### ISSUE TITLE
[Bug] Broken font smoothing/rendering on OnePlus 5T

### ISSUE THREAD

**Issue by yoasif** [mozilla-mobile/fenix#14840]
From https://www.reddit.com/r/firefox/comments/iokv3l/whats_wrong_with_my_default_font_in_firefox_since/

## Steps to reproduce

Looks like this happens on google.com SERPs and site not found (internal) Fenix pages.

### Expected behavior

Good quality font smoothing.

### Actual behavior

![image](https://user-images.githubusercontent.com/152493/92429720-d0c5cd80-f160-11ea-98f3-f0c4c043f570.png)

Text looks bad. 

### Device information

* Android device: OnePlus 5T
* Fenix version: 80.1.2, also beta

**Comment by Dunexus** [comment]
I tried on my OnePlus 5T using latest nightly with both Roboto and OnePlus Slate font and webrender on and off, can't reproduce.
Also, I never seen this on previous versions.

There may be something more to reproduce but I can't figure what.

**Comment by 0yz** [comment]
> There may be something more to reproduce but I can't figure what.

I'm the OP of the Reddit thread. I had some custom fonts in my /system/fonts folder and it seems like Firefox randomly used fonts from that directory. Roboto Thin was one of the fonts and this is what you see on the original screenshot. First i just moved Robto Thin from that folder and then it just took another font from that directory (the font was not roboto and is not assigned anywhere on my device). When i moved everything besides my current system font family (OnePlus Slate) i ended up with the default system **serif** font in Google. My fonts work perfectly fine everywhere else though.

![Screenshot_20200908-190936_Firefox](https://user-images.githubusercontent.com/34684219/92507572-7af83080-f207-11ea-8c3a-1b2b3f335938.png)

**Comment by yoasif** [comment]
Sounds like https://github.com/mozilla-mobile/fenix/issues/5417 

There is a possible workaround listed there - does that work @0yz ?

**Comment by 0yz** [comment]
@yoasif  yes this is exactly the same problem i have. The fonts on the screenshot are the same that i had when i moved fonts from the fonts folder. The fix does work but unfortunately only in Nightly cause about:config is not available in normal Firefox.
Thank you!

**Comment by yoasif** [comment]
@0yz you can use beta, which also has `about:config`.

Closing as a dupe.
